src/components/PokemonList.js

```diff
@@ -3,10 +3,8 @@ import PokemonCard from './PokemonCard';
 import '../styles/PokemonList.scss';
 
 const PokemonList = props => {
-  const {pokemons, searchedName, loading} = props;
-  const filteredPokemons = pokemons
-  .filter(item => item.name.toUpperCase().includes(searchedName.toUpperCase()))
```

Comment:
I extracted the logic of searching to an external component. This way we can test in a separated way and we could replace by a more complex search in case we need to improve the filter.